pkg/target/target.go

```diff
@@ -462,6 +468,20 @@ func (h *K8sValidationTarget) ValidateConstraint(u *unstructured.Unstructured) e
 			return errorList.ToAggregate()
 		}
 	}
+
```

Comment:
This code should live in the webhook, as the list of enforcement actions is not part of the k8s target, but the GK enforcement point.